icons/ghost.svg

```diff
@@ -0,0 +1,16 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M4 10v12l3-3 2 3 3-3 2.5 3 2.5-3 3 3V10" />
+  <path d="M9 10h.01" />
+  <path d="M15 10h.01" />
+  <path d="M20 10a8 8 0 10-16 0" />
```

Comment:
@mittalyashu yeah the wave looked a bit off. because they were not 45 deg. Here is the fix:
```suggestion
  <path d="M9 10h.01" />
  <path d="M15 10h.01" />
  <path d="M12 2a8 8 0 00-8 8v12l3-3 2.5 2.5L12 19l2.5 2.5L17 19l3 3V10a8 8 0 00-8-8z" />
```

Reply:
nice!